backend/src/LibExecution/DvalReprInternalQueryable.fs

```diff
@@ -354,6 +353,7 @@ let parseJsonV0 (types : Types) (typ : TypeReference) (str : string) : Ply<Dval>
                 |> Ply.List.flatten
 
               return DEnum(typeName, caseName, fields)
+          | _, _ -> return Exception.raiseInternal "Unhandled case" []
```

Comment:
maybe something like "Couldn't parse custom type" and include the typeName and valueKind